samples/boards/esp32/wifi_apsta_mode/src/main.cpp

```diff
@@ -0,0 +1,190 @@
+/*
```

Comment:
Question, why the top level main is  a .cpp file and not a regular .c file?

Reply:
Recently, I’ve been working extensively with zephyrOS cpp support. So, I created it the with .cpp. I’ll rename it to .c and disable the cpp support in prj.conf .